tests/functions.py

```diff
@@ -305,3 +309,18 @@ def ping_host(host):
         return False
     else:
         return True
+
+
+def wait_on_job(job_id, max_timeout):
+    global job_results
+    timeout = 0
+    while True:
+        job_results = GET(f'/core/get_jobs/?id={job_id}')
+        job_state = job_results.json()[0]['state']
+        if job_state in ('RUNNING', 'WAITING'):
+            sleep(5)
+        elif job_state in ('SUCCESS', 'FAILED'):
+            return {'state': job_state, 'results': job_results.json()[0]}
+        if timeout == max_timeout:
```

Comment:
If `max_timeout` is not a multiple of `5`, the timeout will never occur. We should use `>=` then instead.

Reply:
True